samples/tab-personal-sso-quickstart/ts/src/components/Tab.tsx

```diff
@@ -154,15 +155,15 @@ class Tab extends React.Component<ITabProps, ITabState> {
   }
   render() {
       let title = this.state.context && Object.keys(this.state.context).length > 0 ?
-        'Congratulations ' + this.state.context?.user?.userPrincipalName + '! This is your tab' : <Loader/>;
+        'Congratulations ' + this.state.context?.user?.userPrincipalName + '! This is your tab' : <Spinner/>;
       let ssoMessage = this.state.ssoToken === "" ?
-        <Loader label='Performing Azure AD single sign-on authentication...'/>: null;
+        <Spinner label='Performing Azure AD single sign-on authentication...'/>: null;
       let serverExchangeMessage = (this.state.ssoToken !== "") && (!this.state.consentRequired) && (this.state.photo==="") ?
-        <Loader label='Exchanging SSO access token for Graph access token...'/> : null;
+        <Spinner label='Exchanging SSO access token for Graph access token...'/> : null;
       let consentMessage = (this.state.consentRequired && !this.state.consentProvided) ?
-        <Loader label='Consent required.'/> : null;
-      let avatar = this.state.photo !== "" ?
-        <Avatar image={this.state.photo} size='largest'/> : null;
+        <Spinner label='Consent required.'/> : null;
+      // let avatar = this.state.photo !== "" ?
+      //   <Avatar image={{src:'this.state.photo'}} size={40}/> : null;
```

Comment:
Why we need this commented code?

Reply:
Avatar is not working that's why we have added spinner.